images/buildpack-golang/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Basic golang buildpack
+
+FROM eu.gcr.io/kyma-project/prow/bootstrap:0.0.1
+
+# Versions
+
+ENV GO_VERSION 1.10.4
+ENV DEP_RELEASE_TAG v0.4.1
+
+# Install Go
+
+RUN wget -q https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz && \
+    tar xzf go${GO_VERSION}.linux-amd64.tar.gz && \
+    rm go${GO_VERSION}.linux-amd64.tar.gz && \
+    mv go /usr/local
+
+ENV GOPATH /workspace/go
+ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+RUN mkdir -p ${GOPATH}/bin
+RUN mkdir -p ${GOPATH}/src
+
+# Install Dep
+
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
```suggestion
RUN curl https://raw.githubusercontent.com/golang/dep/${DEP_RELEASE_TAG}/install.sh | sh
```

Reply:
Dep installation script is using `DEP_RELEASE_TAG` inside do download proper version. Anyway, I will change it to be sure they didn't drop support for it in master.

Reply:
Version `v4.0.1` doesn't have `install.sh` file... I don't want to upgrade `dep`, because it will need changes in projects and it will conflict with Jenkins configuration.